x-pack/plugins/observability/public/components/shared/exploratory_view/header/refresh_button.tsx

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { EuiButton, EuiFlexGroup, EuiFlexItem } from '@elastic/eui';
+import { LastUpdated } from './last_updated';
+import { REFRESH_LABEL } from '../exploratory_view';
+import { useSeriesStorage } from '../hooks/use_series_storage';
+
+export function RefreshButton() {
+  const { setLastRefresh, chartTimeRangeContext } = useSeriesStorage();
+
+  return (
+    <EuiFlexGroup alignItems="center" gutterSize="xs">
```

Comment:
```suggestion
    <EuiFlexGroup alignItems="center" gutterSize="m">
```
I noticed the space between the last updated and refresh button looked a bit too tight, so suggesting to up the gutter size.

**Before**
<img width="387" alt="CleanShot 2022-04-21 at 16 30 33@2x" src="https://user-images.githubusercontent.com/4104278/164481499-8214100a-b309-4edc-b82c-25093545f3b1.png">

**After**
<img width="389" alt="CleanShot 2022-04-21 at 16 33 43@2x" src="https://user-images.githubusercontent.com/4104278/164481571-f1f058ae-6e15-453a-9d1f-dc9de83e7921.png">